src/Telemetry/Events/Event_Subscriber.php

```diff
@@ -101,6 +99,6 @@ public function send_events() {
 		// Get the passed event array.
 		$events = filter_input( INPUT_POST, 'events', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY ); // phpcs:ignore WordPressVIPMinimum.Security.PHPFilterFunctions.RestrictedFilter
 
-		$this->container->get( Event::class )->send_batch( $events );
+		$this->container->get( Event::class )->send_batch( $events ?: [] );
```

Comment:
Upon reflection, I think the better plan is to not send anything if events is empty.

```suggestion
		if ( empty( $events ) ) {
				return;
		}
		
		if ( ! is_array( $events ) ) {
				$events = (array) $events;
		}

		$this->container->get( Event::class )->send_batch( $events );
```